Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerDownloadTask.h

```diff
@@ -99,7 +99,8 @@ class ServiceWorkerDownloadTask : public NetworkDataTask, private FunctionDispat
     Ref<NetworkProcess> m_networkProcess;
     RefPtr<SandboxExtension> m_sandboxExtension;
     FileSystem::PlatformFileHandle m_downloadFile { FileSystem::invalidPlatformFileHandle };
-    int64_t m_downloadBytesWritten { 0 };
+    uint64_t m_downloadBytesWritten { 0 };
+    uint64_t m_expectedContentLength { 0 };
```

Comment:
Shouldn't this be std::optional?  It's possible to not be there.

Reply:
0 for m_downloadBytesWritten means that didReceiveData was never called.
This is true for m_expectedContentLength.